node.js/tests/utils.js

```diff
@@ -0,0 +1,24 @@
+// Set Ingestion Key:
+exports.setEnv = (key, value) => {
+    process.env[key] = value;
+};
+
+// Set Test Data Information:
+exports.setData = (options) => {
+    return {
+        awslogs: {
+            data: require('zlib').gzipSync(new Buffer(JSON.stringify({
```

Comment:
Really dislike inlining `require`s.

Reply:
I call it once - therefore, there is inline `require`

Reply:
Even if it's called once, I think inline `require`s are bad style.

Reply:
Done!